index.html

```diff
@@ -48,4 +48,178 @@ <h2 class="hex connect">LET’S CONNECT</h2>
                     <a href="#"><img src="images/socials/Vector2.svg" /></a>
                 </li>
             </ul>
-        </section>
\ No newline at end of file
+        </section>
+        <div class="works">
+            <section class="grid-items project-1">
+            <img class="project-img" src="images/projects/project-1.png" alt="Project-1" />
+            <div class="details">
+            <div class="titles">
+            <h3 class="project-title">Tonic</h3>
+            <ul class="canopy">
+            <li>CANOPY</li>
+            <li class="bullets"><span class="dot"></span></li>
+            <li class="o-li">Back End Dev</li>
+            <li class="bullets"><span class="dot"></span></li>
+            <li class="o-li">2015</li>
+            </ul>
+            </div>
+            <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+            </p>
+            <ul class="categories hex">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+            </ul>
+            <div class="see-project">
+            <a href="#"><button type="button" class="hex">See Project</button></a>
```

Comment:
- [ ] I would suggest not nesting `button` tag inside an anchor `<a>` tag. You need  to inclose your `see project` button either in an `<a>` tag OR a`button `tag you don't need both. Buttons are supposed to perform a specific action like submitting a form. When you nest one inside the other, it makes it confusing as to what action you want performed. That is why it is best to not nest a button inside an anchor tag.